tests/smtlib/testfile-push-pop1.err.dolmen.expected

```diff
@@ -0,0 +1,4 @@
+
+unknown
+
+unknown
```

Comment:
Unrelated remark: we should have an `(error "something")` output here.

Reply:
It should be done in a PR that centralizes how errors are handled. Should we open an issue about it?

Reply:
Yeah, this is definitely unrelated to this PR, opening an issue for it is a good idea — I thought you had one on error behavior already but I can't find it?

Reply:
It may have gotten lost in the depth of this repo's madness.

Reply:
Actually it was a Zulip message — opened #851 feel free to assign yourself if you want to tackle it.